### PR TITLE
force app config update in onboarding

### PR DESCRIPTION
**Pull request by whiskey**
## Description
The (cached) app configuration was not 'reset' on app reset. This PR fixes this by force-loading the app configuration on first usage in onboarding. The configuration is loaded, regardless the current cache state.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3978

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift [hunk 1]
```diff
@@ -145,7 +145,7 @@ final class OnboardingInfoViewController: UIViewController {
 	}
 
 	private func loadCountryList() {
-		appConfigurationProvider.appConfiguration().sink { [weak self] configuration in
+		appConfigurationProvider.appConfiguration(forceFetch: true).sink { [weak self] configuration in
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:148]: Can we add a comment here on why we need to force fetch?

**Review by nickguendling — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift (hunk 1 above)
Comment (whiskey) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:150]: Sure. I'd like to get a step further and find (later™) a way to make sure that the first access to the app config is ensured to be forced. Currently this bug will reappear if we add something like `appConfigurationProvider.appConfiguration()` before this part of the code is called.

For time reasons I didn't check this in depth.

Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:150]: Could you be so kind a create a refactor story for this, so we don't loose track of this issue? :)

Comment (whiskey) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift:150]: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4060

**Review by marcussc — APPROVED** [review]

**Pull request merged**